pyspecProcScripts/calc_error.py

```diff
@@ -0,0 +1,131 @@
+import pyspecdata as psd
+from .simple_functions import select_pathway
+import numpy as np
+
+
+# {{{ Functions to calculate var and mean of nan-masked data
+#     Pulled from time_domain_noise.py example
+def masked_mean_multi(x, axis=None):
+    "Calculate the mean of nan-masked data on a 1D axis"
+    assert axis is not None
+
+    def masked_mean(x):
+        return np.mean(x[np.isfinite(x)])
+
+    return np.apply_along_axis(masked_mean, axis, x)
+
+
+def masked_var_multi(x, var_has_imag=False, axis=None):
+    "Calculates the variance along a 1D axis"
```

Comment:
it's also important that when you  feed this complex data, it gives you the variance of the real data (or variance of the imaginary data)

Reply:
(I'm saying add this info to the docstring)

Reply:
not resolved yet